packages/components/pages/f-mfa/src/components/Mfa.vue

```diff
@@ -207,18 +243,60 @@ export default {
             console.log('DEBUG - onShowHelpInfo'); // eslint-disable-line no-console
         },
 
-        // To be replaced by API call
-        postValidateMfaToken () {
-            const err = new Error('TEST - 400 error');
-            err.response = {
-                status: 400
-            };
-            throw err;
+        /**
+        * If no error has already occurred it reads, decodes, trims, validates the query
+        * string parameters based on the regex provided. Then store the value against the
+        * supplied property indicated by the key.
+        * If the key is not present in the query string or the value is not valid, the
+        * showErrorPage flag is raised and the issue is logged.
+        *
+        * @param {string} key - The key to search for in the query string.
+        * @param {string} field - The data field to store the value against.
+        * @param {string} regex - The regex to validate the value against.
+        */
+        processQueryString (key, field, regex) {
+            if (this.showErrorPage === true) {
```

Comment:
here you can just do the below unless there some other value that comes back which you're trying to avoid?

`if (this.showErrorPage === true) return;`

Reply:
Not a fan of inline `if`'s as they can make it harder to read.